pkg/securitypolicy/regopolicy_test.go

```diff
@@ -1968,6 +1968,55 @@ func Test_EnforceRuntimeLogging_Not_Allowed(t *testing.T) {
 	}
 }
 
+func Test_Rego_Scratch_Mount_Policy(t *testing.T) {
```

Comment:
Kudos for your continued inclusion of test modules. Unit tests should be considered a requirement for all new code, and you consistently are a role model for this habit.